src/main/java/com/stripe/net/LiveStripeResponseGetter.java

```diff
@@ -321,6 +340,12 @@ static String createQuery(Map<String, Object> params)
 		String charge;
 	}
 
+	private static class OAuthError {
```

Comment:
I think a short comment on this one indicating that it's used to receive an error before converting it to a more suitable user-facing exception would be helpful.

Reply:
:+1: